tool/travis.sh

```diff
@@ -12,7 +12,7 @@ if [ "$#" == "0" ]; then
 fi
 
 pushd $PKG
-pub get || exit $?
+pub upgrade || exit $?
```

Comment:
Please revert this, let's just use the versions in `pubspec.lock`.

Reply:
Can we do this in a way that is maintained by `mono_repo`

I would like to avoid modifying auto-generated files..

Reply:
but fine... let's just do it, we can cleanup CI later :)

Reply:
It is not just the CI vs auto-generated files, but also the baked-in versions we use in prod vs. the versions we are testing against. E.g. in the past we needed to pin `analyzer` to specific versions, and I think it is better to stick with the `pubspec.lock` versions and do a controlled upgrade. Such an issue could be hidden if we blindly do `pub upgrade`.